doc/source/datm.rst

```diff
@@ -121,6 +121,12 @@ DATM%CLM_QIAN
    - streams: CLM_QIAN.Solar,CLM_QIAN.Precip,CLM_QIAN.TPQW
    - datamode: CLMNCEP
 
+DATM%CLMCRUJRA2024
+   - CRUJRA atm input data (CESM I compsets)
+   - datm_mode: CLMCRUJRA2024
+   - streams: CLMCRUJRA2024.Solar,CLMCRUJRA2024.Precip,CLMCRUJRA2024.TPQW
+   - datamode: CLMCRUJRA2024
```

Comment:
datamode here should be CLMNCEP, since this is the general setting and NOT the specific setting.

Reply:
Judging from other examples in this file, I will change both datamode and datm_mode to CLMNCEP.